cypress/integration/apps/citas/agendas/revision-agenda/revision_agendas_con_paciente.spec.js

```diff
@@ -18,32 +18,65 @@ context('CITAS - Revisión de Agendas', () => {
         cy.login('30643636', 'asd').then(t => {
             token = t;
             return cy.createPaciente('paciente-masculino', token);
-        }).then(xhr => {
-            paciente = xhr.body;
-            pacienteDoc = xhr.body.documento;
+        }).then(xhrPac => {
+            paciente = xhrPac.body;
+            pacienteDoc = xhrPac.body.documento;
             return cy.createAgenda('agenda-auditada-con-paciente', null, null, null, token);
-        }).then((xhr) => {
-            idAgenda = xhr.body.id;
-            idBloque = xhr.body.bloques[0].id;
-            idTurno = xhr.body.bloques[0].turnos[1].id;
-            horaInicio = xhr.horaInicio;
+        }).then((xhrAgenda) => {
+            // cy.log(xhrAgenda);
+            idAgenda = xhrAgenda.body.id;
+            idBloque = xhrAgenda.body.bloques[0].id;
+            idTurno = xhrAgenda.body.bloques[0].turnos[0].id;
+            horaInicio = xhrAgenda.horaInicio;
         });
     });
 
     it('Se quita asistencia a paciente existente', () => {
+
+        cy.server();
+        cy.route('GET', '**/api/modules/turnos/agenda/**').as('agenda');
+        cy.route('GET', '**/api/core/mpi/pacientes/**').as('paciente');
+        cy.route('PUT', '**/api/modules/turnos/turno/*/bloque/*/agenda/**').as('putTurno');
+
         cy.goto(`/citas/revision_agenda/${idAgenda}`, token);
         cy.get('tbody:nth-child(1) tr:first-child').click();
         cy.plexSelectType('label="Asistencia"').click().get('.option').contains('No Asistio').click();
         cy.toast('success', 'El estado de la agenda fue actualizado');
+
+        cy.wait('@agenda').then(xhrAgenda => {
+            cy.expect(xhrAgenda.status).to.be.eq(200);
+            const turnoAgendaAsistio = xhrAgenda.response.body.bloques.find(x => x.id === idBloque).turnos.find(y => y.id === idTurno);
+
+            cy.wait('@paciente').then(xhrPaciente => {
+                cy.expect(xhrPaciente.status).to.be.eq(200);
+                cy.expect(xhrPaciente.response.body.id).to.be.eq(turnoAgendaAsistio.paciente.id);
+                cy.expect(turnoAgendaAsistio.asistencia).to.be.eq('asistio');
+            });
+
+            cy.wait('@putTurno').then(xhrTurno => {
+                // cy.log(xhrTurno.response.body);
+                const turnoAgendaNoAsistio = xhrTurno.response.body.bloques.find(x => x.id === idBloque).turnos.find(y => y.id === idTurno);
+
+                cy.expect(xhrTurno.status).to.be.eq(200);
+                cy.expect(turnoAgendaNoAsistio.paciente.id).to.be.eq(turnoAgenda.paciente.id);
```

Comment:
Simplemente corriendo el test me sale:

turnoAgenda is not defined.

por favor revisar!

Reply:
jaja si... renombré la variable a último momento y metí la pata